detectable/src/main/java/com/synopsys/integration/detectable/detectables/pipenv/parse/PipfileLockParser.java

```diff
@@ -0,0 +1,37 @@
+package com.synopsys.integration.detectable.detectables.pipenv.parse;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.jetbrains.annotations.Nullable;
+
+import com.synopsys.integration.detectable.detectable.util.EnumListFilter;
+import com.synopsys.integration.detectable.detectables.pipenv.parse.model.PipfileLock;
+import com.synopsys.integration.detectable.detectables.pipenv.parse.model.PipfileLockDependency;
+import com.synopsys.integration.detectable.detectables.pipenv.parse.model.PipfileLockDependencyEntry;
+
+public class PipfileLockParser {
+    public List<PipfileLockDependency> parse(PipfileLock pipfileLock, EnumListFilter<PipenvDependencyType> dependencyTypeFilter) {
+        List<PipfileLockDependency> dependencies = new LinkedList<>();
+        dependencies.addAll(parseDependencyInfo(pipfileLock.dependencies));
+        if (!dependencyTypeFilter.shouldExclude(PipenvDependencyType.DEV)) {
```

Comment:
```suggestion
        if (dependencyTypeFilter.shouldInclude(PipenvDependencyType.DEV)) {
```
The API of the filter can inverse it for you. Just reads a little cleaner.